src/Messages/Traits/ArrayAccessTrait.php

```diff
@@ -12,6 +12,7 @@ trait ArrayAccessTrait
     /**
      * {@inheritdoc}
      */
+    #[\ReturnTypeWillChange]
```

Comment:
Didn't catch any errors without this change. Is it really needed?

Reply:
See reply to @nagyad below.